recipe/build.sh

```diff
@@ -26,12 +26,18 @@ else
    export ADIOS_MPI="--with-mpi=${PREFIX}"
 fi
 
+# Fortran bindings
+export ADIOS_FORTRAN="--enable-fortran"
+if [[ "$CONDA_BUILD_CROSS_COMPILATION" != "1" ]]; then
```

Comment:
```suggestion
if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
```

Reply:
Ui, forgot to add.

I think I can add `-fallow-argument-mismatch` again to get this compiled...